microshift_updating/microshift-update-options.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="microshift-update-options"]
+= Update options with {product-title} and {op-system-bundle}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-update-options
+
+toc::[]
+
+You can upgrade {op-system-ostree-first} with or without updating {product-title-first} as long as the two versions are compatible:
+
+include::snippets/microshift-rhde-compatibility-table-snip.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+You can only update {product-title} from one minor version to the next in sequence. For example, you cannot update from 4.14 to 4.16. You must update 4.14 to 4.15.
+====
+
+[id="microshift-update-options-standalone-updates_{context}"]
+== Standalone {product-title} updates
+
+You can update {product-title} without reinstalling the applications you created. {op-system-base-full} or {op-system-ostree} updates are also not required to update {product-title}, as long as the existing operating system is still compatible with the new {product-title}.
+
+{product-title} operates as an in-place upgrade and does not require removal of the previous version. Data backups beyond those required for the usual functioning of your applications are also not required. Storage and networking configurations only need to be updated if you create a new image that requires new configurations.
+
+[NOTE]
+====
+Operators previously installed must be reinstalled using manifests.
+====
+
+[id="microshift-update-options-rpm-ostree-updates"]
+=== RPM-OSTree updates
+You can update {product-title} on an OSTRee system such as {op-system-ostree} by building a new image containing the new version of {product-title}. Provisioning and re-installing applications is not needed with this update type. The Greenboot health check automatically rolls back to a previous healthy system state if the update fails. To begin an OSTRee update, use the procedures in the following documentation:
```

Comment:
Provisioning is an open question. As in, we don't need to do this again, correct? https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.13/html/installing/microshift-embed-in-rpm-ostree#provisioning-a-machine_microshift-embed-in-rpm-ostree
And we don't need to reinstall applications. Question wasn't answered on the google doc, so I tossed the sentence in here so it would float to the top.

Reply:
I'm not sure why applications would be tied to MicroShift updates. Maybe we should schedule a quick call to discuss it.